ddtrace/profiling/collector/stack.pyx

```diff
@@ -447,7 +447,7 @@ class StackCollector(collector.PeriodicCollector):
     max_time_usage_pct = attr.ib(factory=attr_utils.from_env("DD_PROFILING_MAX_TIME_USAGE_PCT", 1, float))
     nframes = attr.ib(factory=attr_utils.from_env("DD_PROFILING_MAX_FRAMES", 64, int))
     ignore_profiler = attr.ib(factory=attr_utils.from_env("DD_PROFILING_IGNORE_PROFILER", False, formats.asbool))
-    endpoint_collection_enabled = attr.ib(factory=attr_utils.from_env("DD_PROFILING_ENDPOINT_COLLECTION_ENABLED", True, formats.asbool))
+    endpoint_collection_enabled = attr.ib(default=None)
```

Comment:
It is not passed in `ddtrace/profiling/profiler.py` when this object is created unless I'm missing something.

Reply:
Is it because it was moved to here? https://github.com/DataDog/dd-trace-py/pull/4840/files#diff-b4fb85d7f34e3186361268619b446a1e08fd4206ccd606c27482227348cb8315R126-R128

Reply:
If this environment variable is not used, it should be removed from the documentation.

https://github.com/DataDog/dd-trace-py/blob/903ac658c98c2edf0fd708ad551fadc4665e5317/docs/configuration.rst?plain=1#L400-L403

Reply:
Sorry, typo: it is *now* passed from `ddtrace/profiling/profiler.py`

Reply:
@majorgreys So the env variable is still used, FWIW.

Reply:
It doesn't look like the `DD_PROFILING_ENDPOINT_COLLECTION_ENABLED` is set explicitly in `ddtrace/profiling/profiler.py` when `StackCollector` is being created (https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/profiling/profiler.py#L205). Doesn't this mean that we should keep this env variable here?

Reply:
@Yun-Kim I'm not sure I fully understand: it is set in this PR: https://github.com/DataDog/dd-trace-py/pull/4840/files#diff-b4fb85d7f34e3186361268619b446a1e08fd4206ccd606c27482227348cb8315R212

Reply:
Ahh thanks for clarifying. @majorgreys I think we don't need to worry about removing this env variable from documentation as it is now being used/checked in `ddtrace/profiling/profiler.py`.